locales/pt-BR/translation.json

```
@@ -91,47 +91,47 @@
         "yes-please": "Sim, por favor"
     },
     "$Calendar": {
-        "leave-by": "Based on the time you arrived today, you should leave by",
-        "all-done": "All done for today. Balance of the day:",
-        "last-day-balance": "Last day used for balance",
+        "leave-by": "Com base na hora em que você chegou hoje, você deve sair às",
+        "all-done": "Tudo certo por hoje. Balanço do dia:",
+        "last-day-balance": "Último dia usado para balanço",
```

We have been using `saldo` instead of `balanço`:

```suggestion
        "all-done": "Tudo certo por hoje. Saldo do dia:",
        "last-day-balance": "Último dia usado para o saldo",
```